src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -5259,6 +5259,16 @@ internal static List<CompletionResult> CompleteVariable(CompletionContext contex
 
             var wordToComplete = context.WordToComplete;
             var colon = wordToComplete.IndexOf(':');
+            string scopePrefix;
+            if (colon == -1)
+            {
+                scopePrefix = string.Empty;
+            }
+            else
+            {
+                scopePrefix = wordToComplete.Remove(colon + 1);
+                wordToComplete = wordToComplete.Substring(colon + 1);
+            }
```

Comment:
Please use the same pattern as for wordToComplete:
```c#
var wordToComplete = context.WordToComplete;
string scopePrefix = string.Empty;
var colon = wordToComplete.IndexOf(':');
if (colon >= 0)
{
    scopePrefix = wordToComplete.Remove(colon + 1);
    wordToComplete = wordToComplete.Substring(colon + 1);
}
```

Reply:
I don't personally see it as an improvement, but I've done it anyway. I hope I understood the intent correctly.

Reply:
Thanks! (It is not improvement. It is a style. It makes the code more readable.)